examples/simple_demo/lib/job_queue.ex

```diff
@@ -0,0 +1,36 @@
+defmodule SimpleDemo.JobQueue do
+  require Logger
+  use EctoJob.JobQueue, table_name: "jobs"
+  alias Ecto.Multi
+  require IEx
+
+  @doc "send some payload to test"
+  def enqueue(payload) do
+    job = %{"type" => "test", "payload" => payload}
+
+    Ecto.Multi.new()
+      |> SimpleDemo.JobQueue.enqueue(:some_job, job)
+      |> SimpleDemo.Repo.transaction()
+  end
+
+
+
+  def perform(multi = %Ecto.Multi{}, job = %{"type" => "test", "payload" => payload}) do
```

Comment:
`Ecto.Multi` is aliased at the top of the file, so usage in this code can just be `Multi`